### ISSUE TITLE
Blocks Interfaces with examples breaks assets links

### ISSUE THREAD

**Issue by radames** [gradio-app/gradio#1095]
### Describe the bug

Enabling an interface with `cached_examples` and `examples` using new `Blocks` approaches seems to be not linking the examples correctly . In addition to getting this extra error

```
AttributeError: 'Blocks' object has no attribute 'encrypt'
ERROR:    Exception in ASGI application
```

### Reproduction

Fully functional example
https://colab.research.google.com/drive/1_Ru0M-QS7wdH0PxhKU3a7-mwdxxmd-Hy?usp=sharing

```python

main = gr.Interface(
    fn=classify_image,
    inputs=image,
    outputs=label,
    allow_flagging="never",
    cache_examples=True,
    examples=[
        "gradio/demo/image_classifier/images/cheetah1.jpg",
        "gradio/demo/image_classifier/images/lion.jpg"
        ]
    )

blocks = gr.Blocks()
with blocks:

    md('''
# Testing Blocks
    ''')

    gr.TabbedInterface([main], ["Main"])

    md('''
# Testing Blocks
    ''')
blocks.launch(debug=True, enable_queue=False)
```

### Screenshot

![image](https://user-images.githubusercontent.com/102277/165376263-de687f55-703b-49b1-8853-c26276a3d2c5.png)

![image](https://user-images.githubusercontent.com/102277/165375130-f3856c25-f5eb-4e49-bd9e-d9826dc727a8.png)

### System Info

```shell
gradio==2.9.0b8
```

### Severity

annoyance

**Comment by abidlabs** [comment]
Thanks @radames for catching these issues, let me take a look!

**Comment by abidlabs** [comment]
Just tested and it looks like this is fixed as of `gradio==2.9.0b23`. Thanks a bunch for bringing this to our attention!
